chain/actors/actor_storagemarket.go

```diff
@@ -245,6 +230,7 @@ func (sma StorageMarketActor) AddBalance(act *types.Actor, vmctx types.VMContext
 }
 
 func setMarketBalances(vmctx types.VMContext, nd *hamt.Node, set map[address.Address]StorageParticipantBalance) (cid.Cid, ActorError) {
+	// TODO: iterating over a map might happen in the wrong order, this could have gas implications
```

Comment:
TBH I'd fix this while here, This may bite as pretty soon when people start making market deals.